addOns/reports/src/main/zapHomeFiles/reports/high-level-report/Messages.properties

```diff
@@ -0,0 +1,25 @@
+# i18n strings specific to this report template
+report.template.riskSummaryChart = Vulnerability Severity
+report.template.vulnerabilityCountPerSiteChart = Asset Exposure
+report.template.reportDesc = Report Description
+report.template.bugsCountChart = Count of vulnerability occurrences
+report.template.reportTitle = Report Title
+report.template.vulnerabilityImpact = Vulnerability Impact
+report.template.vulnerabilityDescriptions = Vulnerability Descriptions
+report.template.tableNameCol = Name
+report.template.tableImpactCol = Impact
+report.template.mostCommonBug = Most Common Bug
+report.template.mostSevereAlert = Most Severe Alert
+report.template.section.bugsCountChart = No. of bug occurrences chart
```

Comment:
```suggestion
report.template.section.bugsCountChart = No. Of Bug Occurrences Chart
```

Reply:
<https://capitalizemytitle.com/#Chicago> says this is proper, I think it's because of the period. I'm fine forcing lower if we're agreed?

Reply:
Sounds good. Regarding the above comment, vulnerability should start with upper case?

Reply:
Yup, thanks good catch.